server/chat-plugins/lottery.ts

```diff
@@ -248,7 +248,7 @@ export const commands: ChatCommands = {
 		`/lottery create max winners, name, html - creates a new lottery with [name] as the header and [html] as body. Max winners is the amount of people that will win the lottery. Requires # & ~`,
 		`/lottery delete - deletes the current lottery without declaring a winner. Requires # & ~`,
 		`/lottery end - ends the current lottery, declaring a random participant as the winner. Requires # & ~`,
-		`/lottery editmarkup html - edits the lottery markup with the provided HTML. Requires # & ~`,
+		`/lottery edit max winners, name, html - edits the lottery markup with the provided HTML. Requires # & ~`,
```

Comment:
```suggestion
		`/lottery edit max winners, name, html - edits the lottery with the provided parameters. Requires # & ~`,
```

Reply:
bump ^

Reply:
I still can't merge this suggestion for some reason.